Library/Homebrew/cmd/outdated.rb

```diff
@@ -28,10 +28,18 @@ def outdated
 
   def print_outdated(formulae)
     verbose = ($stdout.tty? || ARGV.verbose?) && !ARGV.flag?("--quiet")
+    fetch_head = ARGV.fetch_head?
 
-    formulae.select(&:outdated?).each do |f|
+    formulae.select do |f|
+      f.outdated?(:fetch_head => fetch_head)
+    end.each do |f|
```

Comment:
Also this pattern of checking for outdated formulae appears in at least three places in this diff. I wonder if there's a good way to refactor this to avoid repeating this three-line `select` code snippet everywhere.